themes/default/content/blog/pulumi-backstage-plugin/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Backstage Plugin Now Available for Pulumi"
+date: 2023-10-06T15:00:48-04:00
+draft: false
+meta_desc: "Seamlessly integrate and manage your infrastructure with the official Pulumi Backstage plugin."
+meta_image: meta.png
+authors:
+    - engin-diri
+tags:
+- backstage
+- developer-portals
+- features
+
+---
+
+## What is Backstage?
+
```

Comment:
I'd remove this because it's not really needed, and it shows up prominently on the index page as well.

```suggestion
```